### PR TITLE
RDK-35648: Override HDMI Color Depth

### PR DESCRIPTION
**Pull request by acheri988**
Reason for change:
Override HDMI Color Depth
Test Procedure: None
Risks: Low

Change-Id: I9e18de182d516d65b1f794163c238621a08212ed
Signed-off-by:Anooj Cheriyan <Anooj_Cheriyan@comcast.com>

### CODE REVIEW

**Comment by Firdose2109** [comment]
Can one of the admins verify this patch?

File: DisplaySettings/DisplaySettings.json [hunk 1]
```diff
@@ -2243,8 +2248,84 @@
...
+                ]
+            }
+        },
+        "getPreferredColorDepth":{
+            "summary": "Returns the current color depth on the selected video display port.\n \n### Event \n\n No Events.",
+            "params": {
+                "type":"object",
+                "properties": {
+                    "videoDisplay": {
+                        "$ref": "#/definitions/videoDisplay"
```
Comment (apatel859) [inline DisplaySettings/DisplaySettings.json:2280]: persist is missing

File: DisplaySettings/DisplaySettings.json [hunk 2]
```diff
@@ -2243,8 +2248,84 @@
             "result": {
                 "$ref": "#/definitions/result"
             }
+        },
+        "getColorDepthCapabilities": {
+            "summary": "Returns supported STB color depth capabilities.\n \n### Event \n\n No Events.",
```
Comment (apatel859) [inline DisplaySettings/DisplaySettings.json:2253]: Returns supported color depth capabilities.

**Comment by acheri988** [comment]
Review comments fixed: 2523

**Pull request closed without merging**